src/game/server.cpp

```diff
@@ -3384,7 +3384,7 @@ namespace server
                 {
                     prev.erase( prev.begin() + i );
                 }
-                while(prev.size() >= G(maphistory))
+                while(prev.size() >= (size_t)(G(maphistory)))
```

Comment:
Please use C++ casts. You may also just use `decltype` to infer the target type from `size` (but that's totally overkill here, since `size`'s type is well defined and won't ever change unless the class changes, and even then, I'd guarantee it'll never be a narrowing cast).

Reply:
Already done in current last commit